### DISCUSSION TITLE
Update ESPHOME lxc

### DISCUSSION THREAD

**Discussion by pbabin2001** [tteck/Proxmox#2998] [Help Needed] [answered]
Folks, 

I'm currently running ESPHOME inside my Home Assistant AND in an lxc. This is temporary until I transfer all my node to the lxc. The point is I got a notification in Home Assistant that an update is available. How I can update my ESPHOME in the lxc ?

Thanks

**Comment by tteck** [comment, marked as answer]
By following the instructions.

![Screenshot 2024-05-17 3 06 08 AM](https://github.com/tteck/Proxmox/assets/21342670/d4a19b44-80cf-4d3a-93a3-a7499bc96d1c)
